codeclimate-wrapper

```diff
@@ -28,7 +28,7 @@ invalid_docker_host() {
   invalid_setup "invalid DOCKER_HOST=$host, must be unset or unix:///var/run/docker.sock"
 }
 
-if command -v docker-machine > /dev/null 2>&1; then
+if [ "$DOCKER_MACHINE_NAME" ] && command -v docker-machine > /dev/null 2>&1; then
```

Comment:
Good to know. I found it there: http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html
which is the bash man. Will change it to `-n`.